### PR TITLE
docs(addition): add mantis tab & docs & fix links

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](github.com/ComposableFi/env/terraform/github.com/branches.tf) 

Makes review faster:
- [x] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [x] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [ ] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

File: docs/docs/technology/mantis/tools.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Tools
+## Development Progress
+As of November 15, 2023 MANTIS is in mainnet testing. Details of this testing period are as follows:
+
+Composable is running MANTIS and the Composable Virtual Machine (CVM) on mainnet. Osmosis, Neutron, and the Composable Cosmos chain will be accessible for cross-chain operations on the mainnet testing. The flow and components of the testing are as follows:
+- We will create faucets for people to use
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/mantis/tools.md:6]: what faucet on mainnet?

File: docs/docs/technology/mantis/tools.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- There is a place where solvers can connect to the mainnet testing over an RPC
+- There is a place where problems can be submitted by users over an RPC
+
+Additional public details of this testing can be found on [this notion page](https://www.notion.so/composablefoundation/Banksy-Testnet-4-72b7f8317bcf406e9059ae258733eb31), which includes specifications and setup/joining instructions. The code repo is available on [this GitHub page](https://github.com/notional-labs/composable-networks/tree/main/banksy-testnet-4).
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/mantis/tools.md:10]: i do not develop on testnet and cannot, until  somebody maintains IBC connection alive, they dead all the time.

I do mainnet and devnet.

Comment (dzmitry-lahoda) [reply docs/docs/technology/mantis/tools.md:10]: If there are not operating bridges there is no MANTIS. Bridges first, MANTIS second.

People struggle mainnet bridge, so I have doubts to run testnets operational is easy.

Comment (JafarAz) [reply docs/docs/technology/mantis/tools.md:10]: which ibc connection on testnet do you need, we can help get it set up

Comment (JafarAz) [reply docs/docs/technology/mantis/tools.md]: removed this content for now

Comment (dzmitry-lahoda) [reply docs/docs/technology/mantis/tools.md]: for any networks which are subject for mantis. also not only get, but maintain.  and for each restnet indexer i use for pools and gas prices.

**Review by blasrodri — DISMISSED** [review]

File: docs/docs/technology/architecture.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ Composable’s architecture is thoughtfully constructed to actualize this missio
 
 These different components collectively facilitate a complete, end-to-end ecosystem for cross-chain DeFi. In brief, the core components of the technical architecture are as follows:
 
-## [The CVM](cvm.md)
+## [MANTIS](mantis.md)
+Multichain Agnostic Normalized Trust-minimized Intent Settlement (MANTIS) is an ecosystem-agnostic intent settlement framework. It is the culmination of all other elements of our architecture; it uses our trust minimized bridge (and thus the IBC as well as all three of our different chains) as well as the CVM to facilitate cross-chain execution of user intents. As a result, we are able to provide users with a novel, intuitive, and streamlined experience for participating in cross-chain DeFi.
```
Comment (blasrodri) [inline docs/docs/technology/architecture.md:10]: it uses Composable's trust minimized bridges

Comment (JafarAz) [reply docs/docs/technology/architecture.md]: edited

**Review by dzmitry-lahoda — APPROVED** [review]

File: docs/docs/technology/architecture.md [hunk 4]
```diff
@@ -6,7 +6,9 @@ Composable’s architecture is thoughtfully constructed to actualize this missio
 
 These different components collectively facilitate a complete, end-to-end ecosystem for cross-chain DeFi. In brief, the core components of the technical architecture are as follows:
 
-## [The CVM](cvm.md)
+## [MANTIS](mantis.md)
+Multichain Agnostic Normalized Trust-minimized Intent Settlement (MANTIS) is an ecosystem-agnostic intent settlement framework. It is the culmination of all other elements mentioned in this page; it uses Picasso's trust minimized bridge (IBC connections) as well as the CVM to facilitate cross-chain execution of user intents. As a result, users can interact with a novel, intuitive, and streamlined experience for participating in cross-chain DeFi.
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/architecture.md:10]: imho MANTIS should use any chain were there is money flow

File: docs/docs/technology/mantis.md [hunk 5]
```diff
@@ -1,40 +1,33 @@
...
+:::
+## Intents are Inherently Cross-Domain
 
-Distinct blockchain ecosystems, though increasing the solution space for problems, do not actually communicate with each other. There is thus a huge amount of potential loss due to lack of synchronicity between these markets. There is no credibly verifiable way to settle orders between chains, and also settle them at the same time. 
+Composable's thesis introduces an additional feature to how intents are currently recognised, interoperability - intents are inherently cross-domain. To understand this, the history of the equities markets provides great insights as it became more efficient with execution being broken across different markets. Prior to the introduction of dark pools, which are private liquidity pools, individuals were only able to use select markets:
 
-So, how can the supply chain be optimised so that there is the remote possibility of facilitating this?
+After the introduction of dark pools, which enabled orders to be broken up and split into different markets, the solution space broadened and more earnings opportunities were made available:
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/mantis.md:15]: in blockchain we call privacy, mev resistance and security, not dark? like if dark is bad? :)

**Review by dzmitry-lahoda — APPROVED** [review]

File: docs/docs/technology/mantis.md (hunk 5 above)
Comment (JafarAz) [reply docs/docs/technology/mantis.md:15]: this is referring to dark pools in traditional stock markets, basically private exchanges used for securities trading.

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by blasrodri — APPROVED** [review]

**Pull request merged**